src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Public/NLHandler.cs

```diff
@@ -1,44 +1,44 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.PowerFx.Core.Utils;
-using Microsoft.PowerFx.Intellisense;
-using Microsoft.PowerFx.LanguageServerProtocol.Handlers;
-using Microsoft.PowerFx.LanguageServerProtocol.Protocol;
-using static Microsoft.PowerFx.LanguageServerProtocol.LanguageServer;
-
-namespace Microsoft.PowerFx.LanguageServerProtocol
-{
-    /// <summary>
-    /// Callback handler invoked by <see cref="LanguageServer"/> to handle NL requests. 
-    /// </summary>
-    public class NLHandler
-    {
-        public virtual bool SupportsNL2Fx { get; } = false;
-
-        public virtual bool SupportsFx2NL { get; } = false;
-
-        public virtual Task<CustomNL2FxResult> NL2FxAsync(NL2FxParameters request, CancellationToken cancel)
-        {
-            throw new NotImplementedException();
-        }
-
-        [Obsolete("Call overload with Fx2NLParameters")]
-        public virtual Task<CustomFx2NLResult> Fx2NLAsync(CheckResult check, CancellationToken cancel)
-        {
-            throw new NotImplementedException();
-        }
-        
-        public virtual async Task<CustomFx2NLResult> Fx2NLAsync(CheckResult check, Fx2NLParameters hints, CancellationToken cancel)
-        {
-#pragma warning disable CS0618 // Type or member is obsolete
-            return await Fx2NLAsync(check, cancel).ConfigureAwait(false);
-#pragma warning restore CS0618 // Type or member is obsolete
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Intellisense;
+using Microsoft.PowerFx.LanguageServerProtocol.Handlers;
+using Microsoft.PowerFx.LanguageServerProtocol.Protocol;
+using static Microsoft.PowerFx.LanguageServerProtocol.LanguageServer;
+
+namespace Microsoft.PowerFx.LanguageServerProtocol
+{
+    /// <summary>
+    /// Callback handler invoked by <see cref="LanguageServer"/> to handle NL requests. 
+    /// </summary>
+    public class NLHandler
+    {
+        public virtual bool SupportsNL2Fx { get; } = false;
+
+        public virtual bool SupportsFx2NL { get; } = false;
+
+        public virtual Task<CustomNL2FxResult> NL2FxAsync(NL2FxParameters request, CancellationToken cancel)
+        {
+            throw new NotImplementedException();
+        }
+
+        [Obsolete("Call overload with Fx2NLParameters")]

```

Comment:
hard to understand changes in this file. please clean up

Reply:
The only change is adding
ScalarRange Range  to Fx2NLParameters. Not sure why it does this weirdness here in this file.